src/snowflake/connector/gcs_util.py

```diff
@@ -6,10 +6,14 @@
 
 import json
 import os
+from collections import namedtuple
 from logging import getLogger
+from typing import Any, Dict
 
 import requests
 
+from snowflake.connector.compat import quote
```

Comment:
I know I suggested this, but I was mistaken import this like: `from .compat import quote` but then run `fix_lint` locally first.
I'm sorry for giving you a wrong suggestion previously!